draft-irtf-cfrg-opaque.md

```diff
@@ -916,24 +915,26 @@ Upon completion of this function, the client MUST send `record` to the server.
 The server stores the `record` object as the credential file for each client
 along with the associated `credential_identifier` and `client_identity` (if
 different). Note that the values `oprf_seed` and `server_private_key` from the
-server's setup phase must also be persisted. The `oprf_seed` value SHOULD be used
+server's setup phase must persist for the online authentication phase. The
+`oprf_seed` value SHOULD be used
 for all clients; see {{preventing-client-enumeration}}. The `server_private_key`
 may be unique for each client.
 
-# Online Authenticated Key Exchange {#online-phase}
+# Online Authentication {#online-phase}
```

Comment:
```suggestion
# Online Authenticated Key Exchange {#online-phase}
```
One output is a session key, after all.

Reply:
My understanding of the draft is that during Online Authentication two tasks happen, and one of them is an Authenticated Key Exchange (AKE). However, Online Authentication, I think is a more general title for describing this big phase, later there is one section only of AKE.

Reply:
OPAQUE is fundamentally a key exchange protocol, so the existing seems more accurate.

Reply:
I am talking about the `Online Authentication phase`, which is a combination of several primitives, one of them is AKE (which seems to be the most important, but not the only one).

Reply:
After talked offline with Chris Wood, we agreed the current version is ok. So, I'm reverting.